site/content/en/docs/getting_started/overview.md

```diff
@@ -0,0 +1,202 @@
+| [**Command Line Tool**](/docs/api_sdk/cli/)                                                    | Self-hosted           | This tool offers a straightforward command line interface for managing CVAT tasks. Currently featuring basic functionalities, it has the potential to develop into a more advanced administration tool for CVAT.                                                    |
+| [**XML Annotation Format**](/docs/manual/advanced/xml_format/)                                 | Cloud and Self-hosted | Detailed documentation on the XML format used for annotations in CVAT essential for understanding data structure and compatibility.                                                                                                                                 |
+| [**AWS Deployment Guide**](/docs/administration/basics/aws-deployment-guide/)                  | Self-hosted           | A step-by-step guide for deploying CVAT on Amazon Web Services, covering all necessary procedures and tips.                                                                                                                                                         |
+| [**Frequently Asked Questions**](/cvat/docs/faq/)                                              | Cloud and Self-hosted | This section addresses common queries and provides helpful answers and insights about using CVAT.                                                                                                                                                                   |
+
+<!--lint enable maximum-line-length-->
+
+## Integrations
+
+CVAT is a global tool, trusted and utilized by teams worldwide.
+Below is a list of key companies that contribute significantly to our
+product support or are an integral part of our ecosystem.
+
+> **Note:** If you're using CVAT, we'd love to
+> hear from you at [contact@cvat.ai](mailto:contact+github@cvat.ai).
+
+- [**Human Protocol**](https://hmt.ai): This platform incorporates CVAT
+  for augmenting its annotation services within the Human Protocol framework.
+- [**FiftyOne**](https://fiftyone.ai): As an open-source tool for dataset
+  management and model analysis in computer vision,
+  FiftyOne is [closely integrated](https://voxel51.com/docs/fiftyone/integrations/cvat.html)
+  with CVAT, enhancing annotation capabilities and label refinement.
+- [**Toloka**](https://toloka.ai): Toloka leverages CVAT for its crowdsourced data labeling
+  and annotation services, enriching its
+  diverse task handling capabilities. For mre information, see [**Integration with Toloka**](/docs/integration/toloka/)
```

Comment:
Probably it is good to add information about Hugging Face and Roboflow, but the primary question how to combine CVAT.ai Cloud and GitHub documentation in one place.